obsercer_pattern.java

```diff
@@ -0,0 +1,75 @@
+import java.util.ArrayList;
```

Comment:
can you please name this file as observer_pattern.java

Reply:
@Sumit-tech-joshi Thank You for let me know.